src/core/store.ts

```diff
@@ -223,9 +224,13 @@ export const createVirtualStore = (
             ([index, size]) => cache._sizes[index] !== size
           );
           // Skip if all items are cached and not updated
-          if (!updated.length) {
+          if (
+            !updated.length &&
+            !_initialRendered
+            ) {
```

Comment:
> I don't know what is the real problem, but re-render solve the issue. Specifically, bypassing cache condition when executing an action for ACTION_ITEM_RESIZE on initial render, and it cause re-render.
However, VList scroll restoration works properly without this workaround🤔.

I investigated why this happens and I found that it is caused by wrong subscribing order.
So I'll revert this change here and commit the fix.